pkg/querier/querier_test.go

```diff
@@ -176,22 +198,63 @@ func TestQuerierWith(t *testing.T) {
 	}
 	distributor.On("QueryStream", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(&unorderedResponse, nil)
 
-	queryable, _, _ := New(cfg, overrides, distributor, queryables, purger.NewNoopTombstonesLoader(), nil, log.NewNopLogger())
+	tCases := []struct {
+		name                 string
+		distributorQueryable QueryableWithFilter
+		queriables           []QueryableWithFilter
```

Comment:
Would you mind renaming `queriables` to `storeQueryables` or something similar? I found it quite hard for me to know the difference between the two queryables at the beginning

Reply:
done